lib/ops/resize/kernel.glsl

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license MIT
+ * @author Arkadiy Pilguk(apilguk@gmail.com)
+ * @author Mihail Zachepilo(mihailzachepilo@gmail.com)
+ * Copyright 2018 Peculiar Ventures and Pentatonica.
+ * All rights reserved.
+ */
+
+vec4 operation(float y, float x) {
+  vec4 value = vec4(0.0);
+
+  for (float dx = 0.0; dx < ceil(H); dx += 1.0) {
+    for (float dy = 0.0; dy < ceil(W); dy += 1.0) {
+      vec4 v = pickValue_tSrc((y * H) + dy, (x * W) + dx);
+
+      if (S == 0.0) {
```

Comment:
I wonder how, but this change fixes the point 1 of main comment.
```suggestion
      if (S == 0.0 && value.r == 0.0) {
```

We probably need to dig in this way to the source of the problem, for example, this one could pretend to be a solution:
```suggestion
      if (S == 0.0 && v.r + v.g + v.b >= value.r + value.g + value.b) {
```